### DISCUSSION TITLE
Whenever I am trying to run my FastAPI code I am getting an error certainly after making all required changes in the code for pydanticV2 . The error is "value error : set is unhashable type " please tell me what changes need to be made in the code or any version conflict ?

### DISCUSSION THREAD

**Discussion by followvipin** [fastapi/fastapi#11278] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
#the error is not giving me any reference of the code.
```

### Description

File "C:\Python312\Lib\site-packages\pydantic\json_schema.py", line 2235, in _deduplicate_schemas
  return list({_make_json_hashable(schema): schema for schema in schemas}.values())
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: unhashable type: 'set' 

### Operating System

Windows

### FastAPI Version

0.101

### Pydantic Version

2.6

### Python Version

3.12

**Comment by YuriiMotov** [comment]
Can you show a minimal code that reproduces this issue, so that we can run it and see the error?

**Comment by YuriiMotov** [comment]
Closing this. Feel free to reopen it whenever you are ready to provide more info
